bundler/spec/dependabot/bundler/update_checker/latest_version_finder_spec.rb

```diff
@@ -439,8 +390,7 @@
     end
```

Comment:
Looks like there's an example above that references the old let: https://github.com/dependabot/dependabot-core/blob/d328bdcb625fb2783bfeb2b42e6febb2617e5989/bundler/spec/dependabot/bundler/update_checker/latest_version_finder_spec.rb#L221-L223

Reply:
👍🏻  Deleted and replaced with a fixture

Reply:
👍🏻  Deleted and replaced with a fixture